cypress/integration/visitorCanNavigateBetweenSPAViews.feature.js

```diff
@@ -54,7 +54,7 @@ describe("visitor can navigate between views", () => {
 
   describe("Search View", () => {
     before(() => {
-      cy.visit("/search");
+      cy.visit("/search_self_care");
```

Comment:
Okay, I'm not sure if I'm okay with this path. Is there any reason why we want to break the REST convention in this path?

Reply:
changed